api/.env

```diff
@@ -53,3 +53,11 @@ JWT_SECRET_KEY=%kernel.project_dir%/config/jwt/private.pem
 JWT_PUBLIC_KEY=%kernel.project_dir%/config/jwt/public.pem
 JWT_PASSPHRASE=6521faf03b5be499f97cf0b40dc495f2
 ###< lexik/jwt-authentication-bundle ###
+
+###> league/oauth2-google ###
+OAUTH_GOOGLE_CLIENT_ID="889440431087-ueuhpadf2g7h5ucdke92mvfaf4l779m4.apps.googleusercontent.com"
```

Comment:
Diese Secrets sollten nicht eingecheckt werden. Gehören in ein .env.local was von git ignoriert wird

Reply:
Habe ich mich auch gefragt aber, dann kurzerhand an https://github.com/ecamp/ecamp3/blob/b34f3cd612e3273a7aa4a6ffb96358a44ece37b3/backend/config/autoload/auth.global.php orientiert ... sind ja dev secrets, wie kommen die auf die Maschine eines neuen eCamp contributors?

Reply:
Wow okay... Es ist natürlich weit weniger kritisch als bei Produktiven Credentials. Dennoch: Beim Qualix haben wir nur eine MiData-Anbindung, und erwarten dort von neuen Entwicklern dass sie die Secrets aus der Testumgebung holen, also schon wissen was sie tun.
Für neue eCamp Contributors ist das hier wohl ein Abwägen zwischen Developer Convenience und "ein gutes Vorbild abgeben", zwischen "unsere MiData-Anbindung kannst du gleich selber ausprobieren" und "nur erfahrene Leute können an der OAuth Anbindung mitarbeiten".

Bin gerade auch nicht sicher was besser ist. Im Zweifelsfall lässt dus mal so, bis zu dem Zeitpunkt an dem wir mal einen Security Scanner über unseren Code laufen lassen.

Reply:
#2034